src/DIRAC/Core/Utilities/DEncode.py

```diff
@@ -44,11 +43,9 @@ def _ord(char):
 # part of the HTTPS transition.
 class types(object):
     IntType = int
-    LongType = int
     FloatType = float
     BooleanType = bool
     StringType = str
-    UnicodeType = type(u"")
```

Comment:
I think this is still needed until v8.1 so that Python 2 based v7r3 clients can still communicate with the server.